src/Core/Services/Implementations/UserService.cs

```diff
@@ -827,6 +827,31 @@ public async Task<IdentityResult> UpdateTempPasswordAsync(User user, string newM
         return IdentityResult.Success;
     }
 
+    public async Task<IdentityResult> UpdateTdeOffboardingPasswordAsync(User user, string newMasterPassword, string key, string hint)
```

Comment:
This should probably be [its own command](https://contributing.bitwarden.com/architecture/server/)

Reply:
Done